pkg/stanza/operator/input/windows/xml.go

```diff
@@ -196,6 +202,77 @@ type EventDataEntry struct {
 	Value string `xml:",chardata"`
 }
 
+type AnyXML struct {
+	tag        string
+	attributes map[string]string
+	chardata   string
+	children   []AnyXML
+}
```

Comment:
The type looks good - I'm not sure about the naming and visibility of the fields.